benchmark/opperf/nd_operations/nn_activation_operators.py

```diff
@@ -55,55 +57,106 @@ def run_activation_operators_benchmarks(ctx=mx.cpu(), dtype='float32', profiler=
-                                                      runs=runs)
+        # Sigmoid => Covered as part of Unary ops
+        # Hard_Sigmoid
+        hard_sigmoid_benchmark_res = run_performance_test([getattr(MX_OP_MODULE, "hard_sigmoid")],
+                                                          run_backward=True,
+                                                          dtype=dtype,
+                                                          ctx=ctx,
+                                                          profiler=profiler,
+                                                          inputs=[{"data": (2**16, 2**16), "alpha": 0.25, "beta": 0.5},
+                                                                  {"data": (2**4, 2**28), "alpha": 0.25, "beta": 0.5},
+                                                                  {"data": (4, 2**30), "alpha": 0.25, "beta": 0.5}
+                                                                 ],
+                                                          warmup=warmup,
+                                                          runs=runs)
 
-    # Softmax, LogSoftmax
-    softmax_benchmark_res = run_performance_test([getattr(MX_OP_MODULE, "softmax"),
-                                                  getattr(MX_OP_MODULE, "log_softmax")],
-                                                 run_backward=True,
-                                                 dtype=dtype,
-                                                 ctx=ctx,
-                                                 profiler=profiler,
-                                                 inputs=[{"data": (1024, 1024), "axis": -1, "temperature": 0.5},
-                                                         {"data": (10000, 1), "axis": -1, "temperature": 0.5},
-                                                         {"data": (10000, 100), "axis": -1, "temperature": 0.5}
+        # Softmax, LogSoftmax
+        softmax_benchmark_res = run_performance_test([getattr(MX_OP_MODULE, "softmax"),
+                                                      getattr(MX_OP_MODULE, "log_softmax")],
+                                                     run_backward=True,
+                                                     dtype=dtype,
+                                                     ctx=ctx,
+                                                     profiler=profiler,
+                                                     inputs=[{"data": (2**16, 2**16), "axis": -1, "temperature": 0.5},
+                                                             {"data": (2**4, 2**28), "axis": -1, "temperature": 0.5},
+                                                             {"data": (4, 2**30), "axis": -1, "temperature": 0.5}
+                                                            ],
+                                                     warmup=warmup,
+                                                     runs=runs)
+    else:
```

Comment:
It seems the only difference between the if and else branch is the `inputs` argument. Can we only generate different inputs in the if/else branch and pass them to the same operator function?